nix/sources.json

```diff
@@ -169,17 +169,17 @@
         "version": "0.11.2"
     },
     "pocket-ic-x86_64-darwin": {
-        "rev": "5849c6daf2037349bd36dcb6e26ce61c2c6570d0",
-        "sha256": "06c547zhdfcf910zfz50g3aniw2b993m7c0rhsn9v1x163xfnzrm",
+        "rev": "6aceb6a35248ef2735ddd9ca99d8a1c6f4a13908",
```

Comment:
This is too old for the PocketIC library v4.0.0. Could you please bump to the PocketIC server v5.0.0 (IC commit cec100d1607eb2258441ff292a144ba88564458e) which is [compatible](https://docs.google.com/document/d/1VYmHUTjrgbzRHtsAyRrI5cj-gWGs7ktTnutPvUMJioU/edit#heading=h.5wf28dvt742x) with the PocketIC library v4.0.0?